libraries/fabric-shim/package.json

```diff
@@ -37,6 +37,25 @@
   },
   "types": "./types/index.d.ts",
   "license": "Apache-2.0",
+  "nyc": {
+    "exclude": [
```

Comment:
don't think this stanza is quite right, there's no gulpfile, do you want to exclude cli.js? and index.js too?